README.md

```diff
@@ -67,41 +71,41 @@ You can get `org-roam-bibtex` up and running by pasting the following sexps in y
 ```el
 ;; If you installed via MELPA
 (use-package org-roam-bibtex
-  :hook (org-roam-mode . org-roam-bibtex-mode))
+  :hook (org-roam-mode . orb-minor-mode))
```

Comment:
I think the last thing to change is `org-minor-mode` in the doc and we're good to go.

Reply:
Oh, I edited the warning message, but forgot the actual docs.

Reply:
Hehe, no worries. :)